src/plugins/crypto/README.md

```diff
@@ -93,23 +105,18 @@ For a custom Botan development file location set:
 
 ### Mac OS X
 
-Both variants of the plugin compile under Mac OS X "El Capitan" (Version 10.11.3 (15D21)).
-
-For the `crypto_gcrypt` variant download and install either [MacPorts](https://www.macports.org/) or [Homebrew](http://brew.sh/).
-Use one of those tools to download and install `libgcrypt`. If you choose MacPorts, you can set the CMake variables like this:
-
-- *LIBGCRYPT_INCLUDE_DIR* to `/opt/local/include/`
-- *LIBGCRYPT_LIBRARIES* to `/opt/local/lib/libgcrypt.dylib`
-
-The CMake command might look something like:
+All variants of the plugin work under Mac OS Sierra (Version 10.12.3 (16D32)).
 
-    cmake -DLIBGCRYPT_INCLUDE_DIR="/opt/local/include/" -DLIBGCRYPT_LIBRARIES="/opt/local/lib/libgcrypt.dylib" -DPLUGINS="crypto;crypto_gcrypt;" /path_to_elektra_src
+To set up the build environment on Mac OS Sierra we reccommend using [Homebrew](http://brew.sh/).
```

Comment:
Typo: recommend

And is the ` #ifdef ELEKTRA_CRYPTO_API_GCRYPT` around `infos/status` still necessary?

Otherwise: Thanks for keeping the docu up-to-date!